README.md

```diff
@@ -79,6 +80,39 @@ then add this to your maven configuration:
 </dependencies>
 ```
 
+### Gradle
+For Gradle there are 2 plugins that help you keep from unnecessarily compiling the `dagger-compiler` library into your project. One for POJO projects and another for Android projects. They allow you to add the `dagger-compiler` artifact as an `apt` dependency.
+```groovy
+buildscript {
+  repositories {
+    // For POJO projects and the net.ltgt.gradle:gradle-apt-plugin
+    maven {
+      url "https://plugins.gradle.org/m2/"
+    }
+    // For Android projects and the com.neenbedankt.gradle.plugins:android-apt plugin
+    mavenCentral()
+  }
+  dependencies {
+    // For POJO projects
+    classpath "net.ltgt.gradle:gradle-apt-plugin:0.5"
```

Comment:
Why not use the shorter `plugin` block syntax that combines the repository, dependency, and appy lines into a single line?